### ISSUE TITLE
Uncaught TypeError: Cannot read property 'editorElement' of undefined

### ISSUE THREAD

**Issue by aumars** [gsmcmullin/atom-gdb-debugger#24]
1. Right-click on a line with code.
2. Add a breakpoint.

**Atom**: 1.20.0 x64
**Electron**: 1.6.11
**OS**: linux 4.12.13-1-ARCH
**Thrown From**: [atom-gdb-debugger](https://github.com/gsmcmullin/atom-gdb-debugger) package 0.2.2

### Stack Trace

Uncaught TypeError: Cannot read property 'editorElement' of undefined

```
At /home/pradana/.atom/packages/atom-gdb-debugger/lib/editor-integration.coffee:7

TypeError: Cannot read property 'editorElement' of undefined
    at posFromMouse (/packages/atom-gdb-debugger/lib/editor-integration.coffee:7:15)
    at EditorIntegration.module.exports.EditorIntegration._toggleBreakpoint (/packages/atom-gdb-debugger/lib/editor-integration.coffee:76:21)
    at /packages/atom-gdb-debugger/lib/editor-integration.coffee:1:1)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/usr/lib/atom/src/command-registry.js:265:29)
    at CommandRegistry.handleCommandEvent (/usr/lib/atom/src/command-registry.js:3:59)
    at CommandRegistry.module.exports.CommandRegistry.dispatch (/usr/lib/atom/src/command-registry.js:166:19)
    at AtomEnvironment.module.exports.AtomEnvironment.dispatchContextMenuCommand (/usr/lib/atom/src/atom-environment.js:1338:28)
    at EventEmitter.outerCallback (/usr/lib/atom/src/application-delegate.js:334:25)
    at emitThree (events.js:116:13)
    at EventEmitter.emit (events.js:194:7)
```

### Commands

```
     -4:49.9.0 tree-view:show (atom-workspace.workspace.scrollbars-visible-always)
     -4:32.1.0 atom-gdb-debugger:toggle-panel (ul.list-inline.tab-bar.inset-panel)
     -4:24.9.0 atom-gdb-debugger:configure (ul.list-inline.tab-bar.inset-panel)
     -4:23.1.0 atom-gdb-debugger:connect (atom-workspace.workspace.scrollbars-visible-always.theme-one-dark-syntax.theme-one-dark-ui)
  3x -4:13.1.0 core:move-left (input.hidden-input)
     -4:11.8.0 core:move-right (input.hidden-input)
     -3:58.4.0 command-palette:toggle (input.hidden-input)
  3x -3:49.7.0 core:backspace (input.hidden-input)
     -3:39.1.0 atom-gdb-debugger:toggle-breakpoint (input.hidden-input)
     -3:24.8.0 command-palette:toggle (input.hidden-input)
     -3:18.5.0 core:move-down (input.hidden-input)
     -3:17.8.0 core:confirm (input.hidden-input)
  2x -3:17.8.0 atom-gdb-debugger:toggle-breakpoint (input.hidden-input)
```

### Non-Core Packages

```
atom-beautify 0.30.5 
atom-gdb-debugger 0.2.2 
atom-ide-ui 0.3.3 
autocomplete-clang 0.11.3 
autocomplete-cmake 0.3.1 
build 0.68.0 
busy 0.7.0 
busy-signal 1.4.3 
file-icons 2.1.11 
highlight-selected 0.13.1 
ide-python 0.1.1 
ide-typescript 0.1.6 
intentions 1.1.5 
language-cmake 0.3.0 
language-ini 1.19.0 
linter 2.2.0 
linter-flake8 2.2.1 
linter-gcc 0.7.1 
linter-shellcheck 1.4.6 
linter-ui-default 1.6.10 
minimap 4.29.6 
monokai 0.24.0 
open-recent 5.0.0 
pigments 0.40.2 
platformio-ide-debugger 1.2.4 
platformio-ide-terminal 2.6.0 
tool-bar 1.1.0 
```

---

This is similar to #23 because both of these issues arise from adding a breakpoint via the atom-gdb-debugger UI.

**Comment by gsmcmullin** [comment]
Fixed in 7a4de67d744a5c51a619df6c7801d1d571e1b6bf

**Comment by SparksDing** [comment]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.48.0 x64
**Electron**: 5.0.13
**OS**: Mac OS X 10.15.5
**Thrown From**: [atom-gdb-debugger](https://github.com/gsmcmullin/atom-gdb-debugger) package 0.2.2


### Stack Trace

Uncaught TypeError: Cannot read property 'editorElement' of undefined

```
At /Users/sparksd/.atom/packages/atom-gdb-debugger/lib/editor-integration.coffee:7

TypeError: Cannot read property 'editorElement' of undefined
    at posFromMouse (/packages/atom-gdb-debugger/lib/editor-integration.coffee:7:16)
    at EditorIntegration.module.exports.EditorIntegration._toggleBreakpoint (/packages/atom-gdb-debugger/lib/editor-integration.coffee:76:21)
    at /packages/atom-gdb-debugger/lib/editor-integration.coffee:1:1)
    at CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:349786)
    at CommandRegistry.dispatch (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:348261)
    at AtomEnvironment.dispatchContextMenuCommand (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:1:726039)
    at EventEmitter.t (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:1:734045)
    at EventEmitter.emit (events.js:194:13)
```

### Commands

```
     -7:22.9.0 gcc-make-run:compile-run (div.line.cursor-line)
  2x -7:01.1.0 gcc-make-run:run-options (div.highlights)
     -4:20.6.0 gcc-make-run:compile-run (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-one-dark-syntax.theme-one-dark-ui)
     -4:20.6.0 core:confirm (input.hidden-input)
     -4:06.5.0 gcc-make-run:run-options (div.line)
     -3:52.8.0 atom-gdb-debugger:toggle-breakpoint (div.line.cursor-line)
  3x -3:37.6.0 deprecation-cop:view (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-one-dark-syntax.theme-one-dark-ui)
     -0:34.7.0 atom-gdb-debugger:toggle-breakpoint (div.line.cursor-line)
```

### Non-Core Packages

```
atom-gdb-debugger 0.2.2 
build 0.70.0 
busy-signal 2.0.1 
gcc-make-run 0.2.12 
intentions 1.1.5 
linter 2.0.0 
linter-clang 4.1.2 
linter-gcc 0.9.0 
linter-ui-default 1.8.1 
```
